### ISSUE TITLE
[NTR] Add CL terms for thymus ASCT+B table

### ISSUE THREAD

**Issue by paolaroncaglia** [obophenotype/cell-ontology#1279]
cc @dosumis and @emquardokus 
As discussed at yesterday’s ASCT+B harmonisation meeting, I’ll be adding CL terms needed for the thymus ASCT+B table.
Ellen has collected information and suggestions from experts in a spreadsheet.
- [x] I’ll review to address any potential issue (e.g. should any term go in Uberon instead; remove redundant superclasses). 
- [x] I’ll add the annotation property ‘present in taxon’ ‘Homo sapiens’. (Done for all terms except the 3 DN thymocytes)
- [x] As these terms are immune-related, I might ask Alex to review my PRs (pointing him to the spreadsheet, mentioning that the gene + protein biomarkers will be in HubMap, not CL, and asking if he'd like to suggest synonyms to aid in text searches). 
- [x] I'll add the new CL IDs in the spreadsheet when committed (already done for the first 12 terms from [this PR](https://github.com/obophenotype/cell-ontology/pull/1297/commits/1f5757ca1cb2845a7ad816ef5b743dfb650bbd51)).
- [x] Push for a CL release?

For reference, the spreadsheet is [here](https://docs.google.com/spreadsheets/d/1BihrOU9vWrsA_wYjcbdPUr42_Sb22CO3gkBDieiYQPs/edit#gid=725346151), and the broader ticket about ASCT+B thymus anatomy work is [here](https://github.com/obophenotype/CCF_tools/issues/29).

**Comment by paolaroncaglia** [comment]
Note for self: following up on double negative thymocyte terms, here are the relevant minutes from the May 19th 2021 CL eds meeting: 
"Taxon constraints (again)
We now have better doc, but some guidance is needed on when to add taxon-specific classes:
General agreement that taxon specific general classes with no taxon specific differentia are out, e.g. we don't add "lymphocyte, human"
We do add taxon specific classes where definition require taxon specific marker 
Are there cases in between these two?
Examples from immune system and brain.
In this case we could reasonably have a more species general term (+ a present in taxon link). 
Note - we have some inconsistency in naming here - "(Mus musculus" vs ", mouse" used elsewhere - but common names are harder to support with automated naming from patterns using NCBI taxonomy.)"
(for reference, full doc with images [here](https://docs.google.com/document/d/1XvMbNvr0FEsdqGhg79BYCYEHSqUxRHMcvhbGizEAht8/edit#heading=h.dy31w9o663n8))

**Comment by paolaroncaglia** [comment]
So I would
- [ ] Edit the existing DN2/3/4 thymocyte terms by renaming them from e.g. 'DN2 thymocyte' to 'DN2 thymocyte (Mus musculus', adding 'present in taxon Mus musculus', and adding rdfs:comments "Molecular markers that characterize double negative thymocytes are different in mouse versus human thymus.";
- [ ] Add new terms for human specified/committed/rearranging double negative thymocytes, with labels, taxon rules and rdfs comments analogous to the mouse terms above.

There already is a broad CL term for 'double negative thymocyte' so we should be good.
@addiehl @emquardokus @dosumis I'll do this unless I hear you screaming ;-)

**Comment by paolaroncaglia** [comment]
Trying to figure out if existing DN2/3/4 thymocyte terms are used to annotate human data elsewhere... will look closer into Ontobee info e.g. for [DN2 thymocyte](http://www.ontobee.org/ontology/CL?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCL_0000806)

**Comment by paolaroncaglia** [comment]
Update, following discussion offline with David and Ellen:
- [ ] I'll leave the existing DN2/3/4 thymocyte terms as they are, without restricting them to mouse, because they don't contain any taxon information, and it's hard to rule out if anyone anywhere have used them for human data. But I'll add an rdfs:comment to each of them "There is growing evidence that molecular markers characterizing double negative thymocytes are different across species, especially in mouse versus human thymus. If in doubt, please consider using CL:0002489 'double negative thymocyte'."
- [ ] I'll add new terms for human specified/committed/rearranging double negative thymocytes, with labels 'x (Homo sapiens)', annotation property ‘present in taxon’ ‘Homo sapiens’, and rdfs:comments "There is growing evidence that molecular markers characterizing double negative thymocytes are different across species, especially in human versus mouse thymus. This term is restricted for use with human data."
- [ ] I'll add the new CL IDs in the spreadsheet
- [ ] Push for a CL release?

**Comment by paolaroncaglia** [comment]
Notes for self: re-do the PR, remembering that there can only be one rdfs:comment per term.

**Comment by paolaroncaglia** [comment]
Hi @dosumis and @emquardokus ,
I added all terms requested for the thymus ASCT+B table (15 CL terms and 1 Uberon term).
Their IDs are in the [spreadsheet](https://docs.google.com/spreadsheets/d/1BihrOU9vWrsA_wYjcbdPUr42_Sb22CO3gkBDieiYQPs/edit#gid=725346151).
All terms are now live in Uberon and CL, except for 3 double-negative thymocyte terms which I just added. There was a CL release 2 days ago, so it's up to you if you wish to push for another CL release at some point, if you deem it necessary before submission. Thanks.
Paola
